packages/lesspass-pure/src/i18n/en.json

```diff
@@ -51,7 +51,7 @@
   "SiteLoginMasterPasswordMandatory": "Site, login, and master password fields are mandatory.",
   "SorryCopy": "Sorry, copying only works in modern browsers.",
   "UpdateYourSearch": "Please try broadening your search.",
-  "Username": "Username",
```

Comment:
Don't remove this key, it's used on other pages

Reply:
Oops good catch, I guess I screwed up with the conflicts I had when I rebased on `master` :grimacing: